### ISSUE TITLE
Calling netlify sites inside ~ causes error

### ISSUE THREAD

**Issue by diwu1989** [netlify/cli#32]
When i run `netlify sites` inside my home ~ directory, I get this error
```
diwu@burnt ~ » netlify sites
fs.js:732
  var r = binding.read(fd, buffer, offset, length, position);
                  ^

Error: EISDIR: illegal operation on a directory, read
    at Error (native)
    at Object.fs.readSync (fs.js:732:19)
    at tryReadSync (fs.js:487:20)
    at Object.fs.readFileSync (fs.js:535:19)
    at readLocalConfig (/usr/local/lib/node_modules/netlify-cli/lib/settings/config.js:18:26)
    at newConfig (/usr/local/lib/node_modules/netlify-cli/lib/settings/config.js:129:15)
    at Command.<anonymous> (/usr/local/lib/node_modules/netlify-cli/lib/settings/config.js:150:5)
    at Command.listener (/usr/local/lib/node_modules/netlify-cli/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:106:13)
    at Command.emit (events.js:191:7)
```

Inside any other directory, it seems to work just fine.

**Comment by loicteixeira** [comment]
I ran into the same issue. This is because `netlify` first saves the *global* config at `~/.netlify/config` (so `~/.netlify/` is a directory) and then try to save the *local* config at `~/.netlify` (as a file). Unfortunately, as you can see [the paths are currently hardcoded](https://github.com/netlify/netlify-cli/blob/bad0690b35a4e5ec6fd124bfe7ad694785ffce78/lib/settings/config.js#L12-L14).

**Comment by bcomnes** [comment]
This should be addressed in the upcoming 2.0.0 release.
